app/controllers/report_controller/reports/editor.rb

```diff
@@ -597,6 +597,8 @@ def gfv_chargeback
         @edit[:new][:cb_tag_cat] = params[:cb_tag_cat]
         @edit[:cb_tags] = entries_hash(params[:cb_tag_cat])
       end
+    elsif params.key?(:cb_without_metric_rollups)
+      @edit[:new][:cb_without_metric_rollups] = params[:cb_without_metric_rollups] == "true" ? true : false
```

Comment:
`params[:cb_without_metric_rollups] == "true" ? true : false` is just `params[:cb_without_metric_rollups] == "true"` ;)